lib/pbench/cli/server/shell.py

```diff
@@ -111,6 +111,7 @@ def run_gunicorn(server_config: PbenchServerConfig, logger: Logger) -> int:
         logger.warning("OpenID Connect client not configured, {}", exc)
         notifier.notify("STOPPING=1")
         notifier.notify("STATUS=OPENID broker not responding")
+        return 1
```

Comment:
Hmm. Yes, the `NotConfigured` exception is specifically because we didn't find the config section or option at all. If it was configured but doesn't respond, we raise a different exception ... which we're not catching here, and will therefore be caught in `main` and reported as an unhandled exception.

While that's not a big deal, it'd be more accurate to reword the `STATUS` to "not configured", or add `OpenIDClient.ServerConnectionTimedOut` here instead of letting it fly free...